homeassistant/components/onkyo/media_player.py

```diff
@@ -265,6 +265,11 @@ def update(self):
                 break
             else:
                 self._current_source = "_".join([i for i in current_source_tuples[1]])
+        self._attributes["preset"] = (
```

Comment:
We don't add the item at all if the value is `None`.

Reply:
What if user change from radio to other source. Preset will be wrong. Should I delete 'preset' from attributes dict then or set it to None?

Reply:
We should not include incorrect info in state attributes.